src/node_messaging.cc

```diff
@@ -375,9 +375,8 @@ Maybe<bool> Message::Serialize(Environment* env,
     env->isolate_data()->node_allocator()->UnregisterPointer(
         contents.Data(), contents.ByteLength());
 
-    array_buffer_contents_.push_back(
-        MallocedBuffer<char> { static_cast<char*>(contents.Data()),
-                               contents.ByteLength() });
+    array_buffer_contents_.emplace_back(static_cast<char*>(contents.Data()),
+                                        contents.ByteLength());
```

Comment:
This is the only place where I’d say that I’d keep the explicit constructor because it makes it clear that ownership is being transferred here (we should still be able to use `emplace_back`, though). If you stick with the current version, can you make that clear via a comment?

Reply:
I make the suggested change. Can you review again ? thanks.

Reply:
@gengjiawen Thanks, still LGTM!